website/content/api-docs/libraries.mdx

```diff
@@ -36,6 +36,16 @@ $ go get github.com/hashicorp/vault/api
 $ gem install vault
 ```
 
+## C#
+
+- [Vault DotNet Client](https://github.com/hashicorp/vault-client-dotnet)
+
+```shell-session
+# Install-Package HashiCorp.Vault -version "0.1.0-beta"
```

Comment:
None of the other libraries on this page mention a specific version number, so this one probably should not either.